internal/caller/client.go

```diff
@@ -0,0 +1,68 @@
+package caller
+
+import (
+	"github.com/the-maldridge/locksmith/pkg/telephone"
+)
+
+const (
+	clientName = "Caller"
+)
+
+// Factory method for the machine configuration
+var (
+	NativeConfig WGConfig
+	Configurator Configurator
+)
+
+// Client represents the application client.
+type Client struct {
+	status        LocksmithStatus
+	telephone     telephone.Telephone
+	configuration WGConfig
+	company       string
```

Comment:
This appears to be macOS specific, please drop it from this PR and add it in your macOS specific configurator.